src/schema/v2/homeView/sections/InfiniteDiscovery.ts

```diff
@@ -9,6 +9,8 @@ export const InfiniteDiscovery: HomeViewSection = {
   featureFlag: "diamond_home-view-infinite-discovery",
   id: "home-view-section-infinite-discovery",
   requiresAuthentication: true,
+  // TODO: update this to match the first release that can support Infinite Discovery
+  minimumEigenVersion: { major: 8, minor: 59, patch: 0 },
```

Comment:
Here I actually wire this up for infinite disco, but can certainly remove this if that's preferable (it might be!).

Thinking of how this will work in the future — it is a simple thing to add this constraint, but it must be done at the right time, in order to not interfere with development, and to not serve up to users prematurely.